dspace-server-webapp/src/main/java/org/dspace/app/rest/authorization/impl/CanCreateSubmission.java

```diff
@@ -0,0 +1,114 @@
+ * but where an administrator account can submit multiple times.
+ */
+@Component
+@AuthorizationFeatureDocumentation(name = CanCreateSubmission.NAME, 
+    description = "It can be used to verify if a user has the rights to create a new submission in the repository.")
+public class CanCreateSubmission implements AuthorizationFeature{
+    public static final String NAME = "canCreateSubmission";
+
+    @Autowired
+    AuthorizeService authService;
+
+    @Autowired
+    ItemService itemService;
+
+    @Autowired
+    ConfigurationService configurationService;
+
+    @Override
+    public boolean isAuthorized(Context context, BaseObjectRest object) throws SQLException {
+        return this.isUserAuthorized(context);
+    }
+
+    @Override
+    public String[] getSupportedTypes() {
+        return new String[] {
+            CollectionRest.CATEGORY + "." + CollectionRest.NAME,
+            SiteRest.CATEGORY + "." + SiteRest.NAME
+        };
+    }
+
+    /**
+     * Check if the user can make a new submission based on his roles.
+     * 
+     * @param ctx: The current DSpace context.
+     * @return: True if the user can submit, false if not :).
+    */
+    private Boolean isUserAuthorized(Context ctx) throws SQLException {
+        EPerson currentUser = ctx.getCurrentUser();
+        if (currentUser != null) {
+            for (Group group: currentUser.getGroups()){
+                // Two different cases to accept the submission creation:
+                // 1. - The user is an administrator, manager... 
+                // 2. - The user is a student which has no pending submissions
+                if (this.canSubmitMultipleTimes(group.getName()) || (this.numberOfCurrentSubmissionForEperson(ctx, currentUser) < 1)) {
+                    return true;
+                };
+            }
+        }
+        return false;
+    }
+
+    /**
+     * Check if the user can make many submissions.
+     * 
+     * @param searchedValue: A role (a string) of a user.
+     * @return: True if the user can submit multiple times, false if not.
+    */
+    private boolean canSubmitMultipleTimes(String searchedValue) {
+        return this.isPresentForProperty("uclouvain.feature.can_create_submission.permit_all_time", searchedValue);
+    }
+
+    /**
+     * Checks if a string value is present in a list of values coming from a DSpace config.
+     * 
+     * @param configurationPropertyName: A configuration property name to check values from.
+     * @param searchedValue: The value we are searching for.
+     * @return: true if searchedValue equals to one of the value of configurationPropertyName, false otherwise.
+    */
+    private boolean isPresentForProperty(String configurationPropertyName, String searchedValue) {
+        for (String string: this.configurationService.getArrayProperty(configurationPropertyName)){
+            if (string.equals(searchedValue)) return true;
+        };
+        return false;
```

Comment:
`return Arrays.asList(this.configurationService.getArrayProperty(configurationPropertyName)).contains(searchedValue);`